rmgpy/solver/surface.pyx

```diff
@@ -404,6 +437,14 @@ cdef class SurfaceReactor(ReactionSystem):
             else:  # three reactants!! (really?)
                 reaction_rate = k * C[ir[j, 0]] * C[ir[j, 1]] * C[ir[j, 2]]
             k = kr[j]
+            if j in cov_dep:
+                cov_dep_j = cov_dep[j]
+                for smiles, parameters in cov_dep_j.items():
+                    if smiles in current_surface_coverages:
+                        theta = current_surface_coverages[smiles]
+                        if theta >= 0.1:
+                            k *= 10. ** (parameters['a'] * theta) * theta ** parameters['m'] * np.exp(-1 * \
+                                    parameters['E'] * theta / (constants.R * self.T.value_si))
```

Comment:
should this be `parameters[x].value_si`?

Reply:
good catch